tests/models/gemma/test_modeling_gemma.py

```diff
@@ -379,40 +380,6 @@ def test_save_load_fast_init_from_base(self):
     def test_past_key_values_format(self):
         pass
 
-    @require_flash_attn
-    @require_torch_gpu
-    @pytest.mark.flash_attn_test
-    @slow
-    def test_flash_attn_2_generate_padding_right(self):
```

Comment:
This always failed, not sure why it has been added here (probably copy-pasta from mistralà

Reply:
I will leave this for you to take a look (or ping someone else) when you are back

Reply:
oh, it's removed. Sorry to bother